go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kaptinlin/jsonschema
 
-go 1.24.0
+go 1.23.0
```

Comment:
This relaxes version requirement a bit if you don't mind. Reason being in Red Hat Enterprise Linux, Go version is lagging few months behind. Can drop it from the patch if you do not like this.